test/functional/tests/google/pipelines_list.spec.ts.mountebank_fixture.json

```diff
@@ -0,0 +1,2683 @@
+{
+  "protocol": "http",
+  "port": 8084,
+  "stubs": [
+    {
+      "predicates": [
+        {
+          "caseSensitive": true,
+          "deepEquals": {
+            "method": "OPTIONS"
+          },
+          "keyCaseSensitive": true
+        },
+        {
+          "caseSensitive": true,
+          "deepEquals": {
+            "path": "/webhooks/preconfigured"
+          },
+          "keyCaseSensitive": true
+        },
+        {
+          "caseSensitive": true,
+          "deepEquals": {
+            "query": {}
+          },
+          "keyCaseSensitive": true
+        }
+      ],
+      "responses": [
```

Comment:
Is this the recorded traffic from Gate? Is the longer term plan to just keep these in Deck?

Reply:
Yeah that's the recorded traffic from Gate.  I think it makes sense to keep them alongside the tests.  The fixture data is going to be required for anyone who wants to reproduce the tests and who doesn't have the exact same cloud setup that a test's author does.

Reply:
Makes sense, I'm just curious how large these will be as we add tests across providers. I think we can cross that bridge when we get to it.